latch_cli/constants.py

```diff
@@ -4,7 +4,34 @@
 from dataclasses import dataclass
 
 
-@dataclass
+@dataclass(frozen=True)
+class LatchConstants:
+    base_image: str = (
+        "812206152185.dkr.ecr.us-west-2.amazonaws.com/latch-base:ace9-main"
+    )
+
+    mib: int = 2**20
+
+    file_max_size: int = 4 * mib
+
+    file_chunk_size: int = 5 * mib
+
+    pkg_name: str = "latch"
+
+    pkg_ssh_key: str = ".latch/ssh_key"
+
+    pkg_config: str = ".latch/config"
+
+    # todo(aidan): make this aware of the current working directory so that we do not remove useful context
+    ignore_regex = re.compile(
+        r"(?:\.git|\.latch_report\.tar\.gz|traceback\.txt|metadata\.json)$"
```

Comment:
```suggestion
        r"(\.git|\.latch_report\.tar\.gz|traceback\.txt|metadata\.json)$"
```
nit: if you aren't using groups anyway just use the shorter syntax. it's free